utils/theme.js

```diff
@@ -13,7 +13,10 @@ const theme = {
     primaryMuted: '#86C6DE',
     primaryInactive: 'rgba(5, 184, 230, 0.5)',
     primaryMutedInactive: 'rgba(134, 198, 222, 0.5)',
-    selection: '#86C6DE'
+    selection: '#86C6DE',
+    wordClass: '#86C6DE',
+    titleWord: '#05A1DE',
+    translatedWord: '#0AA0DE'
```

Comment:
I think these colors look very similar. I can not see a difference on my screen. Are they supposed to be different, and is it visible on other screens?

Reply:
@adcaelum

Reply:
These colours are similar but nonetheless different; the prototype has them as different colours although they're not readily different. I did want to make sure there are separate colours for titleWord and translatedWord since they could be changed in the future, but we can make them the same colour for now (although still split into titleWord and translatedWord)

Reply:
I think the way we should change colors is to change the color used in markup. For example, if we first decide to use `primary` for the title but later want to use `primaryMuted` for the title we would change what variable it uses.

If we use the theme for a centralized store of colors we only get the small benefit of having all colors in one place. Every time we want to change how a piece of the page looks we would have to change either the view, the theme or both, which leads to low cohesion. If we provide a good set of available colors we only concern about how views look like in the views.

That said, if we don't have enough colors in the theme we should add them, but I think only if they are distinctively different from the existing ones. This blue looks deeper / more saturated to me than the existing blues. What do you think of adding a `saturatedPrimary` or `deepPrimary` and using it instead?

Reply:
How about for the sake of simplicity we just merge them into one blue, say the one for translatedWord, and call it `primaryHighlighted` instead?

Reply:
I think that is a good option!